frontend/src/store/questions/index.js

```diff
@@ -60,8 +60,9 @@ const mutations = {
     state.questionObj.answers = [ans, ...ordered]
   },
   FOCUS_LIKES (state, { likes }) {
+    const oldLikes = state.questionObj.question.likes
     state.questionObj.question.likes = likes
-    // TODO - ALREADY UPVOTED
+    state.questionObj.question.userLiked += likes - oldLikes
```

Comment:
I thought we got userLikes in the object anyway?
This method is prone to races with other users isn't it?

Reply:
Doing a PUT request for likes, just returns the new "likes" value for the object. userLiked is not userLikes, it is what the current user has voted (-1, 0, +1). However it is prone to a race. Perhaps I'll just return the correct "userLiked" value with the "likes" value?

Reply:
Will address if I have time though